### PR TITLE
JSON Instead of Form Data

### LINKED ISSUE #78: Make the server take JSON and the client send JSON
This can be done by using a jQuery filter.

### PR DESCRIPTION
**Pull request by jandjorgensen**
Fixes #78 

@akagomez @James0x57 Could use a hand double-checking that I didn't break any requests--sessions *should* be the only thing that needed attention.

### CODE REVIEW

File: services/sessions.js [hunk 1]
```diff
@@ -40,8 +40,8 @@ var isValidPassword = function(user, password) {
 };
 
 app.get('/services/session', function(req, res) {
-	if (req.user) {
-		res.send({id: req.user.id, user: _.omit(req.user.toJSON(), "password")});
+	if (req.body.user) {
```
Comment (justinbmeyer) [inline services/sessions.js]: why is this necessary?

Comment (jandjorgensen) [inline services/sessions.js]: It isn't. That was leftover from before I put in the filter for post and put. Removed.
